pkg/services/store/kind/registry_test.go

```diff
@@ -18,17 +18,17 @@ func TestKindRegistry(t *testing.T) {
 	for _, k := range registry.GetKinds() {
 		ids = append(ids, k.ID)
 	}
-	require.Equal(t, []string{
-		"dashboard",
-		"folder",
-		"frame",
-		"geojson",
-		"jsonobj",
-		"playlist",
-		"png",
-		"snapshot",
-		"test",
-	}, ids)
```

Comment:
Not sure what the intention of checking this was, but it seems inevitable it's no longer relevant now that the registry is generated